### DISCUSSION TITLE
How to customize lazy.nvim config?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1927] [Q&A] [answered]
I want tu disable the  notify for reload config by code
```
return require("lazy").setup({
  change_detection = {
    notify = false,
  },
})

```

But it warn me lazy.nvim config will not take effect . So how to make it?

**Comment by dpetka2001** [comment, marked as answer]
Change it in `/lua/config/lazy.lua`.
